Tests/eppo/OfflineClientTests.swift

```diff
@@ -125,4 +125,77 @@ final class OfflineClientTests: XCTestCase {
         // Verify the updated assignment
         XCTAssertEqual(updatedAssignment, 3.1415926, "Updated assignment uses variation value.")
     }
+
+    func testGetFlagsConfigurationAndOfflineInitialization() throws {
+        // Initialize first client with test configuration
+        let testJsonString = """
+        {
+          "createdAt": "2024-04-17T19:40:53.716Z",
+          "environment": {
+            "name": "Test"
+          },
+          "flags": {
+            "numeric_flag": {
+              "key": "numeric_flag",
+              "enabled": true,
+              "variationType": "NUMERIC",
+              "variations": {
+                "e": {
+                  "key": "e",
+                  "value": 2.7182818
+                },
+                "pi": {
+                  "key": "pi",
+                  "value": 3.1415926
+                }
+              },
+              "allocations": [
+                {
+                  "key": "rollout",
+                  "doLog": true,
+                  "splits": [
+                    {
+                      "variationKey": "pi",
+                      "shards": []
+                    }
+                  ]
+                }
+              ],
+              "totalShards": 10000
+            }
+          }
+        }
+        """
+
+        eppoClient = EppoClient.initializeOffline(
+            sdkKey: "mock-api-key",
+            assignmentLogger: loggerSpy?.logger,
+            initialConfiguration: try Configuration(
+                flagsConfigurationJson: Data(testJsonString.utf8),
+                obfuscated: false
+            )
+        )
```

Comment:
Wondering if it is better for `getFlagsConfiguration` to return a string. We don't have visibility into how this config is being passed around, and a string representation seems more portable than a `Configuration` object in case it needs to go outside the Swift environment. And we already have this pattern for offline initialization from a string